mis_template_financial_report/migrations/13.0.1.1.0/post-migration.py

```diff
@@ -0,0 +1,14 @@
+# Copyright 2022 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    """We modify the expression keeping the customized changes that each user
+    has defined."""
+    kpi = env.ref("mis_template_financial_report.kpi_profit")
+    kpi.expression = kpi.expression.replace(
```

Comment:
As this record is noupdate=0 record, no modification will be preserved on update.

Reply:
Sorry, I thought it was `noupdate=1`.